napari/utils/perf/_config.py

```diff
@@ -17,11 +17,11 @@
 class PerfmonConfigError(Exception):
     """Error parsing or interpreting config file."""
 
-    def __init__(self, message) -> None:
+    def __init__(self, message: str) -> None:
         self.message = message
```

Comment:
I'd like to leave this PR as just typing the existing implementation instead of changing any of the code, if that's okay?

Reply:
I prefer to improve the codebase instead of just plain typing. 

But let call some other core devs @napari/core-devs. If other has the same opinion as you then I will have no problem with it.

Reply:
I tend to agree; if adding types highlughts an issue, I would rather we leave this PR on hold, open another one where we fix the problem, and come back later!

If there's something more frustrating than fixing a weird old legacy piece of code, is having to do so with mypy screaming at you for things that you're trying to remove ^^'

Reply:
👍 I'll convert this to draft then. Please ping me when it's ready to be reviewed again and I can rebase it.